app/controllers/tickets_controller.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+class TicketsController < ApplicationController
+  before_action :set_ticket, only: %i[show update]
+  before_action :set_participating_seller, only: %i[create]
+
+  # GET /tickets
+  def index
+    @tickets = Ticket.all
+
+    render json: @tickets
```

Comment:
Could you use json_response instead? It's defined in Response to be exactly this

Reply:
sure, makes sense to be consistent, I'll swap it